.vscode/launch.json

```diff
@@ -0,0 +1,23 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Run dev",
```

Comment:
As the file for all configurations sits at the root of the directory, it means it covers all 4 GUIs present in this repository. Thus, I would like for the `name` filed of the configurations to be a bit more specific. For example, the one you added is for InfoLogger so the name should inform the developer about it. 
Perhaps a more sensible name would be: `[ilg] run dev`
`[ilg]` stands for InfoLogger GUI and it is a common label used within the team to refer to this project